app/assets/javascripts/listings/templates/listings.html.slim

```diff
@@ -37,19 +37,18 @@ header.lead-header ng-if="hasEligibilityFilters()"
                 use xlink:href="#i-close"
 
 / Notification: No listings with Open Applications
-section.row ng-if="!openListings.length"
-  .large-12.columns
-    .block.bg-mist.margin-top--2x.text-center
-      figure.block-icon
-        img src="/images/icon-home.png"
-      p.t-delta.t-serif.c-oil.no-margin
-        | No listings currently have open applications.
+section.results-section ng-if="!openListings.length"
+  .row
+    .large-12.columns
+      .short-block.bg-dust
+        h2.t-delta.t-serif.c-oil.no-margin
+          | No listings currently have open applications.
```

Comment:
should we start a practice where if there are untranslated stuff in the template of the feature's we're working on that we create a translation for them?